### ISSUE TITLE
What to document about Camera Input: hands

### ISSUE THREAD

**Issue by zepumph** [phetsims/ratio-and-proportion#512]
@terracoda and I are finding that we are asking quite often what information will need to be given about camera input to our users. We will add stuff here as it comes up.

* Hidden under `cameraInput=hands`
* Load time warning (couple second lag on startup)
* Single hand use
* Only supported for the hands interaction
* `showVideo` can help to calibrate where you are interacting with the webcam.
* Options in the input preferences
* O hand gesture + usage with voicing.
* Not supported offline

**Comment by zepumph** [comment]
Tagging @emily-phet and @BLFiedler if they have any other items to communicate.

**Comment by emily-phet** [comment]
Potentially - tips or nuances related to practical use scenarios, such as:
- An explicit indicator that there are two likely usage scenarios - the vertical and horizontal hand movement. 
- If there are any likely collaborative scenarios that *don't* work well. i can't recall if two people standing next to each other can each contribute a hand or not...I think perhaps they can if their other hands aren't picked up on...?

**Comment by terracoda** [comment]
Yes, two people can each hide a hand use their shown hand to operate the sim.

**Comment by BLFiedler** [comment]
I think we considered restricting handedness via built in MediaPipe, but did not implement it, so either participate can contribute either of their hands. I believe it just looks for whichever hand is left/right-most.

> * `showVideo` can help to calibrate where you are interacting with the webcam.
Just adding some clarification on what calibration means in bullet list below.

Also adding point about performance on low-spec devices.


_Updated list:_

* Hidden under `cameraInput=hands`
* Load time warning (couple second lag on startup)
* Single-hand use
* Only supported for the hands interaction
* `showVideo` can help to calibrate where you are interacting with the webcam.
  * Calibration means setting the range you can move hands up and down by adjusting distance between person and webcam.
* Options in the input preferences
* O hand gesture + usage with voicing.
* Not supported offline
* An explicit indicator that there are two likely usage scenarios - the vertical and horizontal hand movement.
* Collaborative scenarios with up to two hands in the detection window (no handedness, any two hands)
* Performance on different platforms (low-spec CPUs and GPUs will introduce delay/lag, so movements need to be slower)

**Comment by BLFiedler** [comment]
Where would we like to document this? Parts of this should find their way into the teacher tips, but only points relevant to how to integrate into a teacher's classroom (so the collaborative scenarios is a good point and what priority the sim makes of hands in view, as well as performance issues)

**Comment by terracoda** [comment]
Should we sheet explaining all of the above linked in the teacher tips?

**Comment by zepumph** [comment]
>Where would we like to document this?

No strong preference, it seems like something that should be supplementary to the sim, so teacher tips, the research page, maybe both? A blog post somewhere by one of us about our "exciting new research"?

**Comment by emily-phet** [comment]
I think we should do a few things:
1) make a very short video (similar to the sound primer") that goes with this sim. That way any other place we provide info can refer to the video, simplifying some of the text. 
2) A short section in the teacher tips with a bullet list that mentions all of these things. Since it's a new feature I think all of these are relevant, and if we keep it short and sweet and refer to the video I don't think it needs to overwhelm the teacher tips doc. 
3) An email announcement when the sim is published that announces the Voicing feature, and as a slightly less emphasized but still present mention of the computer vision feature, with a reference to the video and teacher tips. 

@BLFiedler - we can chat about the best plan for the video - the new video person may be able to be involved.

**Comment by zepumph** [comment]
Over in https://github.com/phetsims/ratio-and-proportion/issues/527 we realized that a list of camera input features that we don't currently have implemented seems prudent for if we ever come back to this issue. For now, I'll just put it in this comment:

**We punted on this, but for next time we are in RAP:**
* https://github.com/phetsims/ratio-and-proportion/issues/527: Voicing + one-handed interaction.

**Comment by BLFiedler** [comment]
RaP teacher tips needs to be updated with this. I'll tackle this tomorrow or next week at the latest

**Comment by terracoda** [comment]
Unless they are already compete, I can take on items 2 and 3 in https://github.com/phetsims/ratio-and-proportion/issues/512#issuecomment-1244903503

**Comment by terracoda** [comment]
@BLFiedler, please re-assign to me if you would like me to take care of items 2 and 3.

**Comment by terracoda** [comment]
This looks great! 
I have one addition for the last bullet:
- This feature interacts with the Voicing feature. If Voicing is enabled, you can silence the current voiced statement by bringing your index finger and thumb together, forming an O-like gesture with your fingers.

**Comment by terracoda** [comment]
I can see how all that text will look on the prototypes page.

**Comment by terracoda** [comment]
Ok, I added some of this to the website. I think we will still need all the details in the Teacher Tips.

**Comment by BLFiedler** [comment]
Teacher tips updated and uploaded to sim page and google drive.

Linking this to the media update issue to accomplish documentation of Camera Input: Hands on the main page.

https://github.com/phetsims/a11y-research/issues/181#:~:text=%40terracoda%5D-,Camera%20Input%3A%20Hands,-Inclusive%20Feature%20Intro
